modules/channel-web/src/views/lite/main.tsx

```diff
@@ -1,4 +1,5 @@
 import classnames from 'classnames'
+import { isUndefined } from 'lodash'
```

Comment:
This will add the whole lodash library on the channel-web lite bundle, which we do not want. To cherry-pick, you need to use `import isUndefined from 'lodash/isUndefined'`